packages/calcite-components/src/components/chip-group/chip-group.scss

```diff
@@ -6,19 +6,19 @@
   display: flex;
   inline-size: 100%;
   flex-wrap: wrap;
-  gap: theme("spacing.2");
+  gap: var(--calcite-spacing-sm);
```

Comment:
@ashetland @SkyeSeitz this is a small change but one that prevents focus overlap at small scale. For now, this is not a documented public component.